README.md

```diff
@@ -14,6 +14,26 @@ Useful for:
 *NOTE:* As of 1.0.6 works with Sidekiq 4.
 *NOTE:* As of 1.0.8 Locking is atomic (set nx/ex) and will no longer lead to batches that are permalocked and stuck
 
+## Installation
```

Comment:
Moved this section to the top because non-Rails users will have to perform an extra setup